experimental/dds/tree/src/test/HistoryEditFactory.tests.ts

```diff
@@ -120,5 +120,28 @@ describe('revert', () => {
 				).to.be.undefined;
 			});
 		});
+		describe('when reverting the insert/detach of an empty trait', () => {
+			it('when reverting a insert of an empty trait', () => {
```

Comment:
for test titles in an 'it' block, describe what is being accomplished (e.g. 'handles reverting empty inserts').
this way, the full sentence (following the nested describes down to the 'it' block) reads "revert when reverting the insert/detach of an empty trait handles reverting empty inserts'

Reply:
(ditto with the test below)